src/main/java/com/entropyteam/entropay/employees/models/Employee.java

```diff
@@ -257,4 +259,13 @@ public String getLabourEmail() {
     public void setLabourEmail(String labourEmail) {
         this.labourEmail = labourEmail;
     }
+
```

Comment:
minor: we have a wired format here

Reply:
Do you mean the line break on line 262 ? May I have to remove it ?

Reply:
sorry, i mean this: 263 to 269 should be aligned to previous lines
![format](https://github.com/entropy-code/entropay-employees/assets/12630904/d5ed6c5d-bdd5-4f66-b9fa-212aa1c01e25)

Reply:
Ready :+1: